src/subdomains/generic/user/models/user-data/user-data.service.ts

```diff
@@ -534,14 +535,23 @@ export class UserDataService {
     master.users = master.users.concat(slave.users);
     master.accountRelations = master.accountRelations.concat(slave.accountRelations);
     master.relatedAccountRelations = master.relatedAccountRelations.concat(slave.relatedAccountRelations);
+    master.kycSteps = master.kycSteps.concat(slave.kycSteps);
     slave.individualFeeList?.forEach((fee) => !master.individualFeeList?.includes(fee) && master.addFee(fee));
     slave.kycClientList.forEach((kc) => !master.kycClientList.includes(kc) && master.addKycClient(kc));
 
+    // copy all documents
+    await this.documentStorageService.copyFiles(slave.id, master.id);
+
```

Comment:
This should be done asynchronically (with error handling), could take veery long 🤔